route/build/src/standalone/get_basin_runoff.f90

```diff
@@ -319,4 +342,60 @@ SUBROUTINE timeMap_sim_forc(tmap_sim_forc,     & ! inout: time-steps mapping dat
 
  END SUBROUTINE timeMap_sim_forc
 
+
+ ! *********************************************************************
+ ! private subroutine: scale and offset of forcing input
+ ! *********************************************************************
+ SUBROUTINE scale_forcing (forcing_data_in,      & ! inout: array of forcing, runoff, evaporation or precipitation
```

Comment:
consistent format - use small case for type, intent, if, end if, etc.

Reply:
tried to make everything lower case